test/integration/verify/controls/azure_migrate_project_event.rb

```diff
@@ -0,0 +1,11 @@
+resource_group = input(:resource_group, value: '')
+project_name = input(:project_name, value: 'inspec-migrate-integ')
```

Comment:
You can pass this value from variable.tf.All the other resources using same convention.

Reply:
yeah that change is actually already done here https://github.com/inspec/inspec-azure/pull/424/files#diff-b25e8cb262b29f5005a96d0c4a06a3deb77698fa6e11e4f43b21f6f4ad0f45e4R170, once that PR is merged it will be automatically available